tests/services/test_auth.py

```diff
@@ -44,7 +43,7 @@ def test_auth(self):
         audience = "foo_audience"
         issuer = "foo_issuer"
         claims = {"foo": "bar", "aud": audience, "iss": issuer}
-        token = jwt.encode(claims, PRIVATE_KEY, algorithm=ALGORITHMS.RS256)
+        token = jwt.encode(claims, PRIVATE_KEY, algorithm="RS256")
```

Comment:
recommend defining a similar algorithms enum somewhere, so the `"RS256"` value is not repeated, but the corresponding enum entry... Actually does not need to be an enum entry, it could just be a static 'constant' in a class, so you don't need to call the enum entry value

Reply:
you're right, that's the one thing i wish this library had. done